docs/learning/tutorials/index.md

```diff
@@ -1,8 +1,6 @@
----
-browserTabTitle: iTwin.js Tutorials
```

Comment:
@williamkbentley I think this change was on master and this PR pulled it over. Should be do that or should it stay as is?

Reply:
The old page didn't have an H1, so the browser title was generic "iModel.js Documentation" this PR adds a H1, so the `browserTabTitle` frontmatter is no longer needed.

Reply:
ah okay cool, thanks